pulsar-client-cpp/perf/RateLimiter.h

```diff
@@ -50,7 +50,7 @@ RateLimiter::RateLimiter(double rate)
         : interval_(std::chrono::microseconds((long)(1e6 / rate))),
           storedPermits_(0.0),
           maxPermits_(rate),
-          nextFree_() {
```

Comment:
These last two changes look completely unrelated. Add something to the commit message to at least explain them.

Reply:
These are to correct some initialization differences with the compiler/arch combinations. Without these, the simple perf producer was not working on arm with GCC 6.3.

It's mentioned (although briefly) in the description: 

> Fixed some variables initialization